sdk/servicebus/service-bus/test/internal/serviceBusClient.spec.ts

```diff
@@ -587,36 +588,22 @@ describe("ServiceBusClient live tests", () => {
     let errorWasThrown: boolean = false;
 
     const env = getEnvVars();
-    const serviceBusEndpoint = (env.SERVICEBUS_CONNECTION_STRING.match(
-      "Endpoint=sb://((.*).servicebus.(windows.net|usgovcloudapi.net|chinacloudapi.cn))",
-    ) || "")[1];
+    const sbFullQualifiedNamespace = env.SERVICEBUS_FQDN;
```

Comment:
could we standardize the access to this environment variable through the `getFullyQualifiedNamespace` function? this way we don't need to repeat the check for it everywhere.